examples/BBRC2019.ipynb

```diff
@@ -0,0 +1,596 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Benchmark EDF R&D "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import openturns as ot\n",
+    "import numpy as np\n",
+    "import otbenchmark as otb"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import openturns.viewer as otv\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "import pylab as pl"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We present the BBRC test cases using the otbenchmark module"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
```

Comment:
There should be no output here; some method probably generates an unwanted message here.